frontend/packages/console-shared/src/components/SvgPodTooltip.scss

```diff
@@ -0,0 +1,16 @@
+.odc-pod-tooltip {
+  fill: var(--pf-global--BackgroundColor--dark-100);
```

Comment:
Can we align the color of the tooltip with the console tooltips. I believe the background is just black.
Also align the font sizes.
Current looks like this...
![image](https://user-images.githubusercontent.com/14068621/62067786-8b676a80-b202-11e9-8729-4eed87968979.png)

Reply:
![Screenshot from 2019-07-30 19-09-35](https://user-images.githubusercontent.com/2561818/62134490-3577ed00-b2fe-11e9-8b27-0aa80012300d.png)

@christianvogt Now tooltip looks like this.

Reply:
The font is still too big and it's overlapping again the decorator. Could you return it to the size it was previously?

Reply:
Went back to fontSize ```--pf-global--FontSize--sm```.